src/js/components/Calendar/Calendar.js

```diff
@@ -847,21 +846,24 @@ const Calendar = forwardRef(
         if (!children) {
           days.push(
             <CalendarDay
+              activeProp={active}
+              day={day}
               key={day.getTime()}
+              disabledProp={dayDisabled && !!dayDisabled}
               buttonProps={{
                 a11yTitle: day.toDateString(),
-                active: active && active.getTime() === day.getTime(),
-                disabled: dayDisabled && !!dayDisabled,
-                onClick: () => onClick(dateObject),
-                onMouseOver: () => setActive(dateObject),
-                onMouseOut: () => setActive(undefined),
+                onClick:
+                  dayDisabled && !!dayDisabled
```

Comment:
You do this pattern in a lot of places and it could be error prone or a maintenance issue if somewhere else forgets to force it to a boolean. Maybe it would be best to change lines 835-837 to take care of it so you can just use `dayDisabled`
```jsx
        const dayDisabled = !!(
          withinDates(day, normalizeInput(disabled))[0] ||
          (bounds && !betweenDates(day, normalizeInput(bounds))));
```

Reply:
Good call, fixed in latest commit